src/index.html

```diff
@@ -18,5 +19,5 @@
 <script src="//cdnjs.cloudflare.com/ajax/libs/ckeditor/4.4.5/ckeditor.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/speakingurl/2.0.0/speakingurl.min.js"></script>
 <script src="//cdnjs.cloudflare.com/ajax/libs/moment.js/2.10.2/moment.min.js"></script>
-<script src="https://cdn.rawgit.com/Hacker0x01/react-datepicker/e5a67bd4/dist/react-datepicker.min.js"></script>
-<script src="app.js?{{t}}"></script>
```

Comment:
Good catch, I had a problem at some point, removed it to test smth, then forgot about it